### PR TITLE
Fix mobs xp, jxp and hxp calculation

### PR DESCRIPTION
**Pull request by Fizo55**

### CODE REVIEW

File: src/NosCore.Parser/Parsers/NpcMonsterParser.cs [hunk 1]
```diff
@@ -112,7 +111,9 @@ public async Task InsertNpcMonstersAsync(string folder)
                 {nameof(NpcMonsterDto.DarkResistance), chunk => Convert.ToInt16(chunk["ATTRIB"][0][7])},
                 {nameof(NpcMonsterDto.MaxHp), chunk => Convert.ToInt32(chunk["HP/MP"][0][2]) + _basicHp[Level(chunk)]},
                 {nameof(NpcMonsterDto.MaxMp), chunk => Convert.ToInt32(chunk["HP/MP"][0][3]) + Convert.ToByte(chunk["RACE"][0][2]) == 0 ? _basicPrimaryMp[Level(chunk)] : _basicSecondaryMp[Level(chunk)]},
-                {nameof(NpcMonsterDto.Xp), chunk =>  Math.Abs(Convert.ToInt32(chunk["EXP"][0][2]) + _basicXp[Level(chunk)])},
+                {nameof(NpcMonsterDto.Xp), chunk =>  Level(chunk) >= 19 ?
```
Comment (0Lucifer0) [inline src/NosCore.Parser/Parsers/NpcMonsterParser.cs:114]: Please make that a function

Comment (Fizo55) [reply src/NosCore.Parser/Parsers/NpcMonsterParser.cs:114]: Already done in local, I'm fixing jxp aswell that's why it's not pushed rn

File: src/NosCore.Database/Entities/NpcMonster.cs [hunk 2]
```diff
@@ -87,8 +87,6 @@ public NpcMonster()
 
         public byte HeroLevel { get; set; }
 
-        public int HeroXp { get; set; }
```
Comment (Fizo55) [inline src/NosCore.Database/Entities/NpcMonster.cs:90]: On official, there is no data about that in the monster informations, it's working like that :
If a monster is in act 6 - 7 so he can give u some hxp (monsterXp / 25)

Comment (erwan-joly) [reply src/NosCore.Database/Entities/NpcMonster.cs:90]: this is still calculation that should happen on parser time to make it more efficient

Comment (erwan-joly) [reply src/NosCore.Database/Entities/NpcMonster.cs:90]: just set HXP to monsterXp/25

**Pull request closed without merging**